packages/administration/src/Twig/ErrorIdExtension.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\AdministrationBundle\Twig;
+
+use Shopsys\FrameworkBundle\Component\Error\ErrorIdProvider;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class ErrorIdExtension extends AbstractExtension
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Error\ErrorIdProvider $errorIdProvider
+     */
+    public function __construct(
+        private readonly ErrorIdProvider $errorIdProvider,
+    ) {
+    }
+
+    /**
+     * @return \Twig\TwigFunction[]
+     */
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('getErrorId', $this->getErrorId(...)),
```

Comment:
please use snake_case - it's recommended by Symfony/Twig

Reply:
fixed